pages/whats-new/2021-09-01-mailjet.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: "Mailjet Destination"
+date: "2021-09-01"
```

Comment:
Assuming this date goes back to when this was released?

Reply:
Yes, I didn't know when exactly to introduce this, but v6 was released around the end of october.

Reply:
https://github.com/grouparoo/grouparoo/releases/tag/v0.6.0